app-main/public/js/test/jest.setup.js

```diff
@@ -0,0 +1,5 @@
+import Quill from "./quillMock.js";
+document.body.innerHtml = `
```

Comment:
Is this required here? I've removed this markup and the tests seem to run fine.

Reply:
I'm unsure why but after removing this I get an undefined reference to Quill in the quill_init.js file.  I think this is because quill is being loaded into the project on the front end via a CDN so when the tests run in a node environment, the reference is unresolved.  I've just confirmed that removing this code breaks the tests for me.  I've attached the output of running "npm run testCoverage" in my terminal here.  Any insight would be greatly appreciated!

![quill_err](https://user-images.githubusercontent.com/46936775/118692211-cd96dd00-b7d7-11eb-9856-ea7fb9543a60.png)

Reply:
Thanks for sharing your test output!

You were right to create a global Quill mock. Just to confirm, I'm just talking about the innerHtml content not the actual mock i.e.
```
document.body.innerHtml = `
  <div id="editor"></div>
`;
```
Is this the code block you removed?

Reply:
I'm going to get this merged in because this isn't crucial and can be addressed another time 😄

Reply:
Hey, sorry about the confusion, I misunderstood.  The innerHtml appears to be unnecessary and can be removed.  I'd be happy to take care of this just to finalize the work I submitted.  I'll add a new pull request right now.  Thanks again for all the input!